lib/services/front.ts

```diff
@@ -187,6 +187,10 @@ export class FrontService extends Messenger implements ServiceListener, ServiceE
 						archive: false,
 					},
 					subject: details.conversation.subject,
+					// Yes, this is the best place I've found to cite the author of a message for now. SyncBot channels
+					// do not actually use there outgoing triggers anyway, so it doesn't matter *too* much.
+					// But it is still wrong.
+					to: [data.toIds.user],
```

Comment:
![front](https://user-images.githubusercontent.com/20857369/28122071-0eb234a0-6715-11e7-9138-8ddd3d95e0dd.png)
I don't care too much about whether the codebase or the webhooks or whatever can get the author back.  I care about what the UI displays.